libs/utils/android/workloads/youtube.py

```diff
@@ -56,6 +56,8 @@ def run(self, exp_dir, video_url, video_duration_s, collect=''):
         # Force screen in LANDSCAPE mode
         Screen.set_orientation(self.target, portrait=False)
 
+        self.target.execute('monkey -p {} 1'.format(self.package))
```

Comment:
Add a comment reporting the reason, why we want to start youtube without playing the video?!?
Add the same reason in the commit message.